### ISSUE TITLE
[es] errors in pairing verbs not caught

### ISSUE THREAD

**Issue by maco** [languagetool-org/languagetool#7549]
I wrote "olvidé escribir…" and then stared at it for a while thinking it seemed wrong, so looked it up, and yep, need a "de" after "olvidar" if it's before a verb infinitive, but Language Tool didn't tell me that. I experimented some in the Language Tool editor, and I don't see these constructions being caught:

[error -> correction]

* Olvidé escribir -> Olvidé **de** escribir
* Sigo **a** escribiendo -> Sigo escribiendo
* Sigo escribir -> Sigo escribi**endo**
* Quiero **a** escribir -> Quiero escribir

(The seguir ones are also applicable to quedarse.)

**Comment by Eloitor** [comment]
"Olvidé escribir" is correct in spanish. You can take a look at 35.8c here: http://aplica.rae.es/grweb/cgi-bin/v.cgi?i=bMVSQgakQjAVVKoY.

You are correct with the other ones.

**Comment by jaumeortola** [comment]
> "Olvidé escribir" is correct in Spanish.

Yes. That is correct. The verb "olvidarse" (reflexive) does require the prepositions "de":

* Me olvidé escribir -> Me olvidé **de** escribir
